### PR TITLE
docs: Add FreeBSD port

### PR DESCRIPTION
**Pull request by laxul**
I created a list as I will add a pkg when it is available.
Also I added a missing space, please let me know if
you want to change BSD to Other BSD at the same [line](https://github.com/laxul/Cozette/blob/freebsd/README.md?plain=1#L112).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,11 @@ Download the `.dfont` and install it with `Font Book .app`. Both the bitmap
 Grab `CozetteVector.ttf`. If you want to get the bitmap versions to work,
 [follow the instructions from here].
 
-### BSD / Solaris / Haiku /Other
+### FreeBSD
+
+* [ports](https://cgit.freebsd.org/ports/tree/x11-fonts/cozette): `cd /usr/ports/x11-fonts/cozette; make install clean`
+
+### BSD / Solaris / Haiku / Other
```
Comment (slavfox) [inline README.md:112]: Should probably read "Other BSD" now, yeah.

**Review by slavfox — CHANGES_REQUESTED** [review]

**Comment by slavfox** [comment]
Oh, this is delightful, thank you! With the little change in wording to "Other BSD" I'll be happy to merge it. If you'd like to, feel free to add a link to your profile and a mention that you're the maintainer, much like ifreund's is linked in the "Arch" section.

Thank you very much!

File: README.md (hunk 1 above)
Comment (laxul) [reply README.md]: done!

**Comment by laxul** [comment]
I should thank you for your amazing font! I really appreciate all your efforts you have put on it!

**Pull request merged**